From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Sun, 24 Dec 2023 18:49:23 +0100
Subject: do_not_use_version_number_in_system_path

Global installation expects system path to be
something like /usr/share/blender/scripts/
without version numbers.

Contributions by:
 Antonio Ospite <ospite@studenti.unina.it>
---
 intern/ghost/intern/GHOST_SystemPathsUnix.cc | 3 +++
 source/creator/CMakeLists.txt                | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/intern/ghost/intern/GHOST_SystemPathsUnix.cc b/intern/ghost/intern/GHOST_SystemPathsUnix.cc
index 567c56a..8bdfc2a 100644
--- a/intern/ghost/intern/GHOST_SystemPathsUnix.cc
+++ b/intern/ghost/intern/GHOST_SystemPathsUnix.cc
@@ -33,6 +33,9 @@ GHOST_SystemPathsUnix::~GHOST_SystemPathsUnix() {}
 
 const char *GHOST_SystemPathsUnix::getSystemDir(int /*version*/, const char *versionstr) const
 {
+  /* XXX On Debian ignore versionstr when building the system path */
+  versionstr = "";
+
   /* no prefix assumes a portable build which only uses bundled scripts */
   if (static_path) {
     static string system_path = string(static_path) + "/blender/" + versionstr;
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 3fe3416..dbf7191 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -377,7 +377,7 @@ if(UNIX AND NOT APPLE)
       set(TARGETDIR_TEXT ".")
       set(TARGETDIR_LIB "./lib")
     else()
-      set(TARGETDIR_VER "./share/blender/${BLENDER_VERSION}")
+      set(TARGETDIR_VER "./share/blender")
       set(TARGETDIR_TEXT "./share/doc/blender")
     endif()
   endif()
